tools/gfx/cpu/cpu-helper-functions.h

```diff
@@ -6,6 +6,6 @@ namespace gfx
 {
 using namespace Slang;
 
-Result SLANG_MCALL createCPUDevice(const IDevice::Desc* desc, IDevice** outDevice);
+
```

Comment:
Can remove this file.

Reply:
Oops, will remove these and submit whenever the shader cache work is ready for a PR.